.travis.yml

```diff
@@ -0,0 +1,41 @@
+sudo: required
+
+scala:
+  - "2.11.8"
+
+matrix:
+  include:
+    - os: linux
+      jdk: oraclejdk8
+      language: scala
+      before_install:
+        - sudo apt-get -qq update
+        - sudo sh -c "echo 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.7 main' >> /etc/apt/sources.list"
+        - sudo sh -c "echo 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise main' >> /etc/apt/sources.list"
+        - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
+        - sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
+        - sudo apt-get -qq update
+        - sudo apt-get install -y libgc-dev clang++-3.7 llvm-3.7 llvm-3.7-dev llvm-3.7-runtime llvm-3.7-tool
+    - os: osx
+      osx_image: xcode7.3
+      before_install:
+        - brew update
+        - brew install sbt
+        - brew install bdw-gc
+        - brew link bdw-gc
+
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/scala-native
+
+script:
```

Comment:
We also need to check that author of PR signed the CLA: https://github.com/scala-native/scala-native/blob/master/.travis.yml#L39